src/utils.jl

```diff
@@ -25,3 +27,24 @@ function with_temp_depot(f::Function)
     end
     return result
 end
+
+
+"""
+    env_threshold_count(envvar::String, default)
+
+Return an integer (intrerpreted as a number of lines) read
+from the specified environment variable.
+"""
+function env_threshold_count(envvar::String, default)
+        v = get(ENV, envvar, nothing)
+    if v == nothing
+        return default
+    end
+    m = match(r"(?<count>^[0-9]+)$", v)
+    if m != nothing
+        return parse(Int64, m["count"])
+    end
+    @warn "Value $v of environment variable $envvar is not a line count.  Using default of $default"
+    default
+end
```

Comment:
I think we don't need this, since we aren't using ENV variables for these

Reply:
I don't know what you're looking at, but I think this is already gone.  I'll make sure.  I'm prettuly sure I left no changes in src/utils, but moved the remaining generic functions to src/AutoMerge.util.

Reply:
I meant to emove it, but apparently didn't.  Gone now.